pkgs/top-level/all-packages.nix

```diff
@@ -11729,7 +11729,7 @@ with pkgs;
   copper = callPackage ../development/compilers/copper {};
 
   inherit (callPackages ../development/compilers/crystal {
-    llvmPackages = llvmPackages_10;
+    llvmPackages = if stdenv.system == "aarch64-darwin" then llvmPackages else llvmPackages_10;
```

Comment:
Is llvmPackages ahead of llvmPackags_10?

Reply:
It is ahead of llvmPackages_10, yes. (llvmPackages_10 and below are broken on aarch64-darwin.)

What would be the best version to pin it to, in your opinion? I'm guessing llvmPackages_11 would be the best since it's the closest version to llvmPackages_10. (Why do we pin to llvmPackages_10, anyways? Does upstream recommend/only support some versions or something?)

Reply:
We've had some issues in the past with specific llvm versions as we were typically ahead of what crystal upstream would support.